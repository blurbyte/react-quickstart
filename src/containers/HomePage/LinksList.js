import styled from 'styled-components';

const LinksList = styled.ul`
  list-style: none;
  margin-left: 0;
  padding-left: 0;
`;

export default LinksList;
