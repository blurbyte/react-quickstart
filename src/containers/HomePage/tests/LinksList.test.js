import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import LinksList from '../LinksList';

describe('<AppWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <LinksList />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
