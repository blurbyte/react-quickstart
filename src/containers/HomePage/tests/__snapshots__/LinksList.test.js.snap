// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should render correctly 1`] = `
.dPAACO {
  list-style: none;
  margin-left: 0;
  padding-left: 0;
}

<ul
  className="dPAACO"
/>
`;
